dev/com.ibm.jbatch.internal/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
```

Comment:
This should be specifically Java 8 and not just JRE_CONTAINER.
Also the bnd.classpath.container should be in here as well.